foursight_core/react/api/react_routes.py

```diff
@@ -76,6 +85,44 @@ def reactapi_route_header_noenv() -> Response:  # noqa: implicit @staticmethod v
         """
         return app.core.reactapi_header(app.current_request.to_dict(), app.core.get_default_env())
 
+    @route("/{env}/certificates", authorize=False)
+    def reactapi_route_certificates(env: str) -> Response:  # noqa: implicit @staticmethod via @route
+        """
+        Note that this in an UNPROTECTED route.
```

Comment:
I wonder if we should make these operations and the things they call have some token like `public` in their name to remind us of this fact. I haven't though through whether that would get unwieldy, just thinking aloud.